test/functional/page_objects/management/saved_objects_page.ts

```diff
@@ -253,19 +254,21 @@ export function SavedObjectsPageProvider({ getService, getPageObjects }: FtrProv
 
     async getRelationshipFlyout() {
       const rows = await testSubjects.findAll('relationshipsTableRow');
-      return mapAsync(rows, async (row) => {
-        const objectType = await row.findByTestSubject('relationshipsObjectType');
-        const relationship = await row.findByTestSubject('directRelationship');
-        const titleElement = await row.findByTestSubject('relationshipsTitle');
-        const inspectElement = await row.findByTestSubject('relationshipsTableAction-inspect');
-        return {
-          objectType: await objectType.getAttribute('aria-label'),
-          relationship: await relationship.getVisibleText(),
-          titleElement,
-          title: await titleElement.getVisibleText(),
-          inspectElement,
-        };
-      });
+      return await Promise.all(
```

Comment:
Again, we only need to use await within the async function passing to map because we need to wait for each Promise to resolve before we return an object with their resolved values. But when returning from an async function, we could just return the Promise directly without using await.